.gitignore

```diff
@@ -82,5 +82,8 @@ db_data
 es_data
 gem_cache
 
+# Compose generated files
+bundle_finished
```

Comment:
We can leave this as is, but I generally make my sentinel files dotfiles, so they are "hidden".

Reply:
Nice point. Updated to a new path with a hidden file.